recipe/meta.yaml

```diff
@@ -21,14 +20,14 @@ requirements:
     - setuptools
     - wheel
   run:
-    - python >=3.7
+    - python
     # Optional dependencies are numpy, python-lz4, psutil
 
 test:
   requires:
     - pip
-    - python <3.10
-    - pytest
+    - python
+    - pytest  # pytest is needed for some imports
   commands:
     - pip check
   imports:
```

Comment:
Could you add `doc_source_url`: `https://github.com/joblib/joblib/tree/1.1.1/doc` ?

Reply:
And missing a `description`:
```
Joblib is a set of tools to provide lightweight pipelining in Python. In particular: transparent disk-caching of functions and lazy re-evaluation (memoize pattern), easy simple parallel computing.
Joblib is optimized to be fast and robust on large data in particular and has specific optimizations for numpy arrays.
```